components/Deck/ContentPanel/DeckModes/DeckEditPanel/GroupDetailsModal.js

```diff
@@ -51,31 +51,34 @@ class GroupDetailsModal extends React.Component {
     }
 
     render() {
+        let showContent = this.props.group && this.props.group.creator;
+
         let members = [];
         //first the creator
-        let optionalText = (this.props.group.creator.organization || this.props.group.creator.country) ?
+        let optionalText = (showContent && (this.props.group.creator.organization || this.props.group.creator.country)) ?
             (this.props.group.creator.organization || this.context.intl.formatMessage(this.messages.unknownOrganization)) + ', ' + (this.props.group.creator.country || this.context.intl.formatMessage(this.messages.unknownCountry)) :
             '';
-        members.push(
-          (
-            <div className="item" key={this.props.group.creator.userid}>
-              <div className="ui grid">
-                <div className="one wide column middle aligned">
-                  <UserPicture picture={ this.props.group.creator.picture } username={ this.props.group.creator.username } avatar={ true } width= { 24 } />
-                </div>
-                <div className="fifteen wide column">
-                  <TextArea className="sr-only" id={'usernameIsALinkHint' + this.props.group.creator.userid} value={this.context.intl.formatMessage(this.messages.linkHint)} tabIndex ='-1'/>
-                  <a className="header" href={'/user/' + this.props.group.creator.username} target="_blank">{this.props.group.creator.displayName || this.props.group.creator.username}</a>
-                  <div className="description">
-                    {this.context.intl.formatMessage(this.messages.groupCreator)}
+        if (showContent)
+            members.push(
+              (
+                <div className="item" key={this.props.group.creator.userid}>
+                  <div className="ui grid">
```

Comment:
Images may be included as Floating images instead of using a Grid. This would ease the DOM - better browser efficiency